part-spec/power.json

```diff
@@ -697,6 +697,516 @@
+          "description": "typical output current a device can regulate",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "ioutMin": {
+          "description": "minimum output current a device can regulate",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "ioutMax": {
+          "description": "maximum output current a device can regulate",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "ioutAccuracyTyp": {
+          "description": "typical accuracy of output current regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "ioutAccuracyMin": {
+          "description": "minimum accuracy of output current regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "ioutAccuracyMax": {
+          "description": "maximum accuracy of output current regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswTyp": {
+          "description": "typical switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswMin": {
+          "description": "minimum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswMax": {
+          "description": "maximum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentConsumption": {
+          "description": "current consumption of a device",
+          "type": "array",
+          "items": {
+            "$ref": "definitions.json#/currentConsumption"
+          }
+        },
+        "integratedFets": {
+          "description": "whether a device contains integrated switching mosfets",
+          "type": "boolean"
+        },
+        "integratedFetProperties": {
+          "description": "describes integrated fet current limits and rdson properties",
+          "$ref": "#/$defs/powerFetProperities"
+        },
+        "currentMatchingAccuracyTyp": {
+          "description": "typical current matching between LED strings",
+          "comment": "units of percentage",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentMatchingAccuracyMin": {
+          "description": "minimum current matching between LED strings",
+          "comment": "units of percentage",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentMatchingAccuracyMax": {
+          "description": "maximum current matching between LED strings",
+          "comment": "units of percentage",
+          "$ref": "definitions.json#/unit"
+        },
+        "integratedBoost": {
```

Comment:
some overlap with integrated fets here, but I guess this could be just the switching drivers